code/modules/mob/living/say.dm

```diff
@@ -331,20 +331,27 @@ var/global/list/channel_to_radio_key = new
 				var/dst = get_dist(get_turf(M),get_turf(src))
 				var/runechat_enabled = M.client?.is_preference_enabled(/datum/client_preference/runechat_mob)
 
-				if(dst <= message_range || (M.stat == DEAD && !forbid_seeing_deadchat)) //Inside normal message range, or dead with ears (handled in the view proc)
+				var/message_range_boosted = message_range //We have this before the standard stuff so boosted whispers process regularly...
+				var/w_scramble_range_boosted = w_scramble_range //... and so you still can't hear things offscreen.
+				if(ishuman(M))
+					var/mob/living/carbon/human/H = M
+					message_range_boosted = message_range + H.species.hearboost //2023-02-25, var/hearboost on species datum boosts whisper hear range
+					w_scramble_range_boosted = w_scramble_range + H.species.hearboost //for those species with big fuckin chonker ears, see? could later be expanded to boost individuals, too (eg. ling mutation)
+
+				if(dst <= message_range || (dst <= message_range_boosted && whispering) || (M.stat == DEAD && !forbid_seeing_deadchat)) //Inside normal message range, boosted whisper range, or dead with ears (handled in the view proc)
 					if(M.client && !runechat_enabled)
 						var/image/I1 = listening[M] || speech_bubble
 						images_to_clients[I1] |= M.client
 						M << I1
 					M.hear_say(message_pieces, verb, italics, src, speech_sound, sound_vol)
 				if(whispering && !isobserver(M)) //Don't even bother with these unless whispering
-					if(dst > message_range && dst <= w_scramble_range) //Inside whisper scramble range
+					if(dst > message_range_boosted && dst <= w_scramble_range_boosted) //Inside whisper scramble range
 						if(M.client && !runechat_enabled)
 							var/image/I2 = listening[M] || speech_bubble
 							images_to_clients[I2] |= M.client
 							M << I2
 						M.hear_say(stars_all(message_pieces), verb, italics, src, speech_sound, sound_vol*0.2)
-					if(dst > w_scramble_range && dst <= world.view) //Inside whisper 'visible' range
+					if(dst > w_scramble_range_boosted && dst <= world.view) //Inside whisper 'visible' range
 						M.show_message("<span class='game say'><span class='name'>[name]</span> [w_not_heard].</span>", 2)

```

Comment:
```suggestion
				if(dst <= min(message_range + M.hearing_boost_range(), world.view) || (M.stat == DEAD && !forbid_seeing_deadchat)) //Inside normal message range, boosted whisper range, or dead with ears (handled in the view proc)
					if(M.client && !runechat_enabled)
						var/image/I1 = listening[M] || speech_bubble
						images_to_clients[I1] |= M.client
						M << I1
					M.hear_say(message_pieces, verb, italics, src, speech_sound, sound_vol)
				if(whispering && !isobserver(M)) //Don't even bother with these unless whispering
					if(between, dst message_range + M.hearing_boost_range(), w_scramble_range + M.hearing_boost_range()) //Inside whisper scramble range
						if(M.client && !runechat_enabled)
							var/image/I2 = listening[M] || speech_bubble
							images_to_clients[I2] |= M.client
							M << I2
						M.hear_say(stars_all(message_pieces), verb, italics, src, speech_sound, sound_vol*0.2)
					if(between(dst, w_scramble_range + M.hearing_boost_range(), world.view) //Inside whisper 'visible' range
						M.show_message("<span class='game say'><span class='name'>[name]</span> [w_not_heard].</span>", 2)
```
This diff renders really messily but I'm basically proposing you make a proc on /mob to get the "hearing boost range", and override that on human mobs to get the species boost range. Drops the mess of `message_range_boosted` and `w_scramble_range_boosted`.
Check my syntax on `between()`, the order of arguments is out of my ass.